maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java

```diff
@@ -304,23 +320,24 @@ public String toString() {
      * Represents a string in the version item list, usually a qualifier.
      */
     private static class StringItem implements Item {
-        private static final List<String> QUALIFIERS =
-                Arrays.asList("alpha", "beta", "milestone", "rc", "snapshot", "", "sp");
+        // 'pre-release' < 'snapshot' < 'release'
```

Comment:
I don't see how this comment applies to the field. They have different contents.

Reply:
fair i'll remove the comment

Reply:
done